contracts/LightClient.sol

```diff
@@ -121,4 +150,49 @@ contract LightClient is ILightClient, Ownable {
     function proposedValidators() external view override returns (address[] memory, uint256[] memory) {
         return staking.proposedValidators();
     }
+
+    function getEpochIdx(uint256 height) public view override returns (uint256) {
+        require(isInHeightRange(height), "out of height range");
+        //Reduce the times of Sload
+        uint256 _epochPeriod = epochPeriod;
+        return (height + _epochPeriod - 1) / _epochPeriod;
+    }
+
+    function isInHeightRange(uint256 height) public view override returns (bool) {
+        (uint256 min, uint256 max) = heightRange();
+        if (height >= min && height <= max) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    function minEpochIdx() public view override returns (uint256) {
+        if (curEpochIdx >= TOTAL_EPOCH) {
+            return curEpochIdx - TOTAL_EPOCH + 1;
+        }
+        return 0;
+    }
+
+    function heightRange() public view override returns (uint256 min, uint256 max) {
+        min = _minHeight();
```

Comment:
I am not sure the equation is correct - suppose
- `epochPeriod = 10000`
- `curEpochIdx = 4`
- `curEpochHeight = 40000`
- `TOTAL_EPOCH = 2`

Therefore, the range will return `[(3 - 1) * 10000 + 1, 40000 + 10000] = [20001, 50000]`, while the expected should be `[30001, 50000]`?